src/main/java/homework_2/pyramid_printer/PyramidPrinter.java

```diff
@@ -0,0 +1,54 @@
+package homework_2.pyramid_printer;
+
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+public class PyramidPrinter implements Runnable {
+
+    public static final String ERROR_MESSAGE;
+    public static final String WELCOME_MESSAGE;
+    public static final String ENTER_HEIGHT_MESSAGE;
+    public static final String REGEX;
+
+    static {
+        ERROR_MESSAGE = "Only 1 non-negative integer is allowed as passed parameter";
+        WELCOME_MESSAGE = "Welcome to the Pyramid Printer Application";
+        ENTER_HEIGHT_MESSAGE = "Enter the height parameter";
+        REGEX = "\\d+";
```

Comment:
Opt: just instantiate them in fields. Also I'm not sure they should be public

Reply:
The reason I didn't define the values of the variables in the beginning is because I just wanted to practice using a static initialization block.
The only reason I marked the variables as public is because they are final and therefore cannot be changed.
But I will heed your pieces of advice for future tasks. Thanks.